schemes/rayleigh_friction/rayleigh_friction.meta

```diff
@@ -0,0 +1,126 @@
+[ccpp-table-properties]
+  name = rayleigh_friction
+  type = scheme
+
+[ccpp-arg-table]
+  name  = rayleigh_friction_init
+  type  = scheme
+[ pver ]
+  standard_name = vertical_layer_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ raytau0_nl ]
+  standard_name = model_top_decay_time_for_rayleigh_friction
+  long_name = Approx decay time at model top for RF
+  units = days
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ raykrange_nl ]
+  standard_name = number_of_vertical_layers_for_rayleigh_friction
+  long_name = Range of levels with RF applied
+  units = count
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ rayk0_nl ]
+  standard_name = center_vertical_layer_for_rayleigh_friction
+  long_name = Vertical level at which RF profile is centered
+  units = index
+  type = integer
+  dimensions = ()
+  intent = in
+[ masterproc ]
+  standard_name = flag_for_mpi_root
+  units = flag
+  type = logical
+  dimensions = ()
+  intent = in
+[ iulog ]
+  standard_name = log_output_unit
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+[ccpp-arg-table]
+  name  = rayleigh_friction_run
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
```

Comment:
Delete `ncol` from the calling list as no longer needed

Reply:
Did it, caused problems